teuthology/task/install/valgrind.supp

```diff
@@ -445,3 +445,41 @@
     ...
     fun:PyObject_Call
 }
+
+# rados cython constants
+{
+   <insert_a_suppression_name_here>
```

Comment:
@dillaman probably insert a suppression name here as suggested or remove this line (and below)?